taca_ngi_pipeline/cli.py

```diff
@@ -68,15 +68,24 @@ def deliver(ctx, deliverypath, stagingpath, uppnexid, operator, stage_only, forc
 			  envvar='STATUS_DB_CONFIG',
 			  type=click.File('r'),
 			  help='Path to statusdb-configuration')
+@click.option('--order-portal',
+			  default=None,
```

Comment:
Ironic to default none but default is envvar. Aw well, might be click goofiness.